src/handler/store.js

```diff
@@ -1,6 +1,14 @@
-function appStore() {
+function appStore(callback) {
```

Comment:
WDYT of changing this fn name to a more explicit one. For example: `onStoreUpdate`

Reply:
done